### PR TITLE
build: remove obsolete `payload-examples/{ghes-221,ghes-222,ghes-30,github.ae}/integration_installation{,_repositories}` directories

### PR DESCRIPTION
**Pull request by octokitbot**
BREAKING CHANGES: examples for `integration_installation` and `integration_installation_repositories` have been removed for GHES 2.21/2.22/3.0 and GHAE

### CODE REVIEW

**Review by gr2m — APPROVED** [review]
Looks like a re-formatting HTML change without any relevant changes for us. Can you confirm @wolfy1339

File: cache/api.github.com/webhook-events-and-payloads.html [hunk 1]
```diff
@@ -4948,23 +4931,6 @@ <h2 id="installation_repositories">
...
-          p-3
-          color-border-info color-bg-info
-          f5
-        "
-      >
-        <p>
-          <strong>Note:</strong> This event replaces the deprecated
-          <code>integration_installation_repositories</code> event.
-        </p>
-      </div>
```
Comment (gr2m) [inline cache/api.github.com/webhook-events-and-payloads.html:4967]: we do have examples for `integration_installation` and `integration_installation_repositories`. I suggest we remove any reference to these deprecated events from this repository, as they are no longer mentioned in the official docs at https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads

Comment (gr2m) [reply cache/api.github.com/webhook-events-and-payloads.html:4967]: ping @wolfy1339 what do you think?

Comment (wolfy1339) [reply cache/api.github.com/webhook-events-and-payloads.html:4967]: Sounds good
We already removed them for `api.github.com`, only remaining references are GHES and GitHub AE

Comment (gr2m) [reply cache/api.github.com/webhook-events-and-payloads.html:4967]: done via 11fc6a0

**Review by gr2m — APPROVED** [review]

**Pull request merged**